README.md

```diff
@@ -0,0 +1,185 @@
+  to_port           = 80
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = "${aws_security_group.packer.id}"
+}
+```
+
+CloudWatch logs are also configured here
+```
+resource "aws_cloudwatch_log_group" "ami_builder" {
+  name              = "/aws/lambda/ami_builder"
+  retention_in_days = 30
+  tags              = "${local.common_tags}"
+}
+```
+
+Step 3: Create a Lambda Payload File
+In order to configure the behaviour of the Lambda function, e.g. what source AMI to use,
+you will need to provide it with a payload file. This payload file is output as a manifest.json file which is created here.
+[payload file creation](https://github.ucds.io/dip/aws-management-infrastructure/blob/master/ci/meta.yml)
+Please see link for [Packer docs](https://www.packer.io/docs/builders/amazon.html) for further information.
+An example is given below:
+
+```
+            cat << EOF > manifest.json
+            {
+            "log_level":                      "${LOG_LEVEL}",
+            "packer_template_bucket_region":  "${PACKER_TEMPLATE_BUCKET_REGION}",
+            "packer_template_bucket":         "${PACKER_TEMPLATE_BUCKET}",
+            "packer_template_key":            "${PACKER_TEMPLATE_KEY}",
+            "provision_script_bucket_region": "${PROVISION_SCRIPT_BUCKET_REGION}",
+            "provision_script_bucket":        "${PROVISION_SCRIPT_BUCKET}",
+            "provision_script_keys":          ${PROVISION_SCRIPT_KEYS},
+            "provision_file_keys":            ${PROVISION_FILE_KEYS},
+            "source_ami_virt_type":           "${SOURCE_AMI_VIRT_TYPE}",
+            "source_ami_name":                "${SOURCE_AMI_NAME}",
+            "source_ami_root_device_type":    "${SOURCE_AMI_ROOT_DEVICE_TYPE}",
+            "source_ami_owner":               "${SOURCE_AMI_OWNER}",
+            "instance_type":                  "${INSTANCE_TYPE}",
+            "ssh_username":                   "${SSH_USERNAME}",
+            "subnet_id":                      "${SUBNET_ID}",
+            "ami_name":                       "${AMI_NAME}",
+            "profile":                        "${PROFILE}",
+            "security_group_id":              "${SECURITY_GROUP_ID}",
+            "ami_users":                      "${AMI_USERS}",
+            "region":                         "${REGION}"
+            }
+            EOF
+```
+
+Step 4: Create an S3 Bucket for Configuration Items
+As you can see from the payload file above, the Lambda function expects you to
+have an S3 bucket that contains the packer generic configuration template key file and
+(optionally) an S3 bucket containing any provisioning scripts and associated
+deployable artifacts that those scripts reference (the same bucket can be used
+for both, of course)
+This allows the Lambda itself to be completely generic, and to be used to
+generate arbitrarily complex AMIs, providing, of course, that the build can
+complete within the Lambda's defined execution time.
```

Comment:
Probably worth noting that this execution time is configured to be Lambda's maximum, of 15 minutes. We've not had one exceed this yet but eventually we may.